clang/test/CIR/Executables/hello.c

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang -fclangir-enable -fclangir-direct-lowering -o %t %s
```

Comment:
@lanza was this flag needed because codegen for MLIR in-tree dialects is the default? Just wondering if we still need this and if direct-lowering isn't the default, it should, and we'd rename this to tackle the in-tree stuff.

Reply:
Through MLIR is currently the default: https://github.com/llvm/clangir/blob/main/clang/lib/CIR/FrontendAction/CIRGenAction.cpp#L79

We should definitely change that at this point though.